benchmarks/benchmark.py

```diff
@@ -92,6 +92,14 @@ def run_benchmark(exe_filepath, n_repeats=1, csv_out_file=None):
     run_command(command)
 
 
+def pick_color(n):
+    str_bit_reversed_n = '{:015b}'.format(n + 1)[::-1]
+    r = 0.9 * ((int(str_bit_reversed_n[0::3], 2) / 2. ** 5 + 0.3) % 1)
+    g = 0.9 * ((int(str_bit_reversed_n[1::3], 2) / 2. ** 5 + 0.3) % 1)
+    b = 0.9 * ((int(str_bit_reversed_n[2::3], 2) / 2. ** 5 + 0.3) % 1)
+    return r, g, b
```

Comment:
This is one way, but maybe sample from hsl space and then convert to rgb?

Reply:
I tried hsv first, but subjectively I find colors generated this way easier to distinguish. I think the issue is that h,s and v components affect end color in different magnitudes.